server/views/pages/changeEstablishment.njk

```diff
@@ -0,0 +1,45 @@
+{% extends "layout.njk" %}
+{% from "components/card/card.njk" import card %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% set pageHeaderTitle = "Select Establishment" %}
+{% set pageTitle %}
+    {% if errors | length %}Error: {% endif %}{{ applicationName }} - {{ pageHeaderTitle }}
+{% endset %}
+
+{% set backLinkHref = backLink %}
+
+{% set enabledPrisons = [{ prisonId: 'HEI', name: 'Hewell (HMP)'}, { id: 'BLI', name: 'Bristol (HMP)'}] %}
```

Comment:
To make it better ready for the next stage, move this into the actual route and pass it in 👍